plugin/src/main/java/io/jshift/maven/plugin/mojo/build/AbstractJshiftMojo.java

```diff
@@ -44,7 +44,7 @@
 
     // For verbose output
     @Parameter(property = "fabric8.verbose", defaultValue = "false")
```

Comment:
One fabric8 property here, I know it was already there not introduced by this PR but can you fix it ?

Reply:
yes